L/libdrm/bundled/patches/meson_amdgpu.patch

```diff
@@ -0,0 +1,13 @@
+diff --git before/libdrm-2.4.103/amdgpu/meson.build after/libdrm-2.4.103/amdgpu/meson.build
```

Comment:
It'd be good to upstream this

Reply:
https://gitlab.freedesktop.org/mesa/drm/-/merge_requests/100

Reply:
:100: